Mathlib/Topology/Algebra/InfiniteSum/UniformOn.lean

```diff
@@ -208,4 +223,13 @@ theorem HasProdLocallyUniformlyOn.tprod_eqOn [T2Space α]
     (h : HasProdLocallyUniformlyOn f g s) : Set.EqOn (∏' i, f i ·) g s :=
   fun _ hx ↦ (h.hasProd hx).tprod_eq
 
+@[to_additive]
+lemma HasProdLocallyUniformlyOn.tendstoLocallyUniformlyOn_finset_range
```

Comment:
I think strictly speaking this should now be
```suggestion
lemma HasProdLocallyUniformlyOn.tendstoLocallyUniformlyOn_finsetRange
```
but that's inconsistent with `HasProdUniformlyOn.tendstoUniformlyOn_finset_range` (which I don't think you should change here)

Reply:
So are you saying we change it here but not on`HasProdUniformlyOn.tendstoUniformlyOn_finset_range`? why not both?

Reply:
Fair enough, I'll change both occurrences here and leave the wider problem to someone else/another PR.